documents-webapp/src/main/webapp/vue-app/documents/components/body/table/DocumentActionMenu.vue

```diff
@@ -27,7 +31,8 @@ export default {
     mobileOnlyExtensions: ['favorite','details'],
     desktopOnlyExtensions: ['edit'],
     editExtensions: 'edit',
-    fileOnlyExtension: ['download','favorite','visibility']
+    fileOnlyExtension: ['download','favorite','visibility'],
+    fileExtensionsEditable: ['.docx','.pptx','.xlsx']
```

Comment:
Supported OO files are configured inside a service and thus should be retrieved in UI to match the exact supported list IMO. @mkrout WDYT ?

Reply:
+1, the list of supported files should not be hardcoded.